src/app/modules/dialog/components/subsystem-partially-created-dialog/subsystem-partially-created-dialog.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
+import { NvmeOfSubsystem } from 'app/interfaces/nvme-of.interface';
+import { SubsystemPartiallyCreatedDialogComponent } from './subsystem-partially-created-dialog.component';
+
+describe('SubsystemPartiallyCreatedDialogComponent', () => {
+  let spectator: Spectator<SubsystemPartiallyCreatedDialogComponent>;
+  let loader: HarnessLoader;
+
+  const mockSubsystem = {
+    id: 1,
+    name: 'test-subsystem',
+    nqn: 'nqn.2023-01.io.truenas:test-subsystem',
+  } as unknown as NvmeOfSubsystem;
```

Comment:
`unknown` isn't necessary here. It complains because there is no `nqn` field, there is `subnqn`. And also you are not using that field anyway.